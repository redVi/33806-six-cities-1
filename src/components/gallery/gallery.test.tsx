import React from 'react';
import renderer from 'react-test-renderer';
import Gallery from '@/components/gallery/gallery';

describe(`Gallery`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(
      <Gallery images={[`apartment.jpg`, `room.jpg`, `house.jpg`]} maxCount={2} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
