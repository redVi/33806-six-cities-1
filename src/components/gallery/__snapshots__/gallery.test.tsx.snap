// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery renders correctly 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="apartment.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="room.jpg"
      />
    </div>
  </div>
</div>
`;
